src/main/java/attendance/utility/DateTimeParser.java

```diff
@@ -0,0 +1,45 @@
+package attendance.utility;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public final class DateTimeParser {
+
+    private static final String PATTERN = "yyyy-MM-dd HH:mm";
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern(PATTERN);
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```
(반영하지 않으셔도 됩니다.) 상수의 목적은 모든 매직넘버/리터럴을 없앤다기보다는 의미를 잘 전달하기 위함이에요. 그래서 사실 모든 것을 상수화한다기보다는 목적에 따라서 상수화해주시면 좋습니다.

Reply:
의견 감사합니다! 상수의 목적을 인지하고 적용해보겠습니다! 🚀